docs/dev-environment.md

```diff
@@ -5,16 +5,17 @@ title: Setup Dev Environment
 
 ## Setup Backend
 
-Download go dependencies with [golang/dep](https://github.com/golang/dep).
+Download go dependencies with go modules enabled.
 
 ```
-$ dep ensure
+$ export GO111MODULE=on
+$ go get
```

Comment:
Do you know the difference between `go mod download` and `go get`?

Reply:
It's coming from this issue https://github.com/golang/go/issues/26610#issuecomment-408654653

It behaves somewhat like `go get -d` but in this case I think a simple `go get` fits.